From 611093148574164fcf4f24f8c076d09473f655d7 Mon Sep 17 00:00:00 2001
From: Xiao Ni <xni@redhat.com>
Date: Mon, 30 Sep 2019 19:48:00 +0800
Subject: [PATCH] Don't need to check recovery after re-add when no I/O writes
 to raid
Git-commit: 611093148574164fcf4f24f8c076d09473f655d7
Patch-mainline: mdadm-4.1+
References: jsc#SLE-10078, jsc#SLE-9348

If there is no write I/O between removing member disk and re-add it, there is no
recovery after re-adding member disk.

Signed-off-by: Xiao Ni <xni@redhat.com>
Signed-off-by: Jes Sorensen <jsorensen@fb.com>
Signed-off-by: Coly Li <colyli@suse.de>

---
 clustermd_tests/02r1_Manage_re-add | 2 --
 1 file changed, 2 deletions(-)

diff --git a/clustermd_tests/02r1_Manage_re-add b/clustermd_tests/02r1_Manage_re-add
index dd9c416..d0d13e5 100644
--- a/clustermd_tests/02r1_Manage_re-add
+++ b/clustermd_tests/02r1_Manage_re-add
@@ -9,8 +9,6 @@ check all state UU
 check all dmesg
 mdadm --manage $md0 --fail $dev0 --remove $dev0
 mdadm --manage $md0 --re-add $dev0
-check $NODE1 recovery
-check all wait
 check all state UU
 check all dmesg
 stop_md all $md0
-- 
2.25.0

